### ISSUE TITLE
Refactor TexQueryGenerator to be a utility class

### ISSUE THREAD

**Issue by jimmyli97** [TU-Berlin/mathosphere#56]
This should be refactored into a utility class that accepts as parameters: tex string, URL, params. and outputs a custom result class. Maintaining state should not be necessary

**Comment by physikerwelt** [comment]
I don't understand. What's the problem you want to solve?

**Comment by jimmyli97** [comment]
This isn't very important - it's just something I noticed and can be fixed at a later time.

All TexQueryGenerator really does is convert Tex to MathML using a given set of parameters. So following standard java conventions this should be converted into a static class that doesn't store exceptions or results, but instead uses static conversion methods to return results. The results themselves should be made into their own class, and shouldn't be stored alongside the conversion method.

Basically, according to standard object oriented conventions, the actual conversion method should be isolated from the results themselves in order to reduce the number of possible bugs
